### PR TITLE
✨ Add main page

### LINKED ISSUE #34: Main Page
### Screenshot
Bildschirmfoto 2021-06-20 um 09 23 46

### To Do
Add single components to main page
- [ ] `component` Hero to header (fixed)
- [ ] `component`NavBar to footer (fixed)
- [ ] `component` groupcard to main (scrollable)

### PR DESCRIPTION
**Pull request by Dev-dfm**
- Add components to main page
- The filter buttons and the search button at the header are animated by clicking
- The cards are scrollable
- The NavBar in the footer is also animated by clicking the items

Try it out:
https://fit-2-gether-dev-dfm-is-28yrsg.herokuapp.com/storybook/?path=/story/page-main--main-page

fixes #34

### CODE REVIEW

File: src/app/pages/Main/Main.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import NavBar from '../../components/NavBar/NavBar';
+
+export default function Main(): JSX.Element {
+  return (
+    <div className={styles.container}>
+      <header>
+        <Hero />
+      </header>
+      <main className={styles.cards}>
+        <div className={styles.card}>
```
Comment (jujuzitroni) [inline src/app/pages/Main/Main.tsx:14]: I'm not sure if it is really necessary to put the Card components into a div each. Could the className be given to the Card directly?

I imagine you will map the cards into the main later on.

File: src/app/components/Card/Card.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export default function Card({
       <article className={styles.card__article}>
         <section className={styles.card__header}>
           <div className={styles.card__title}>
-            <h2>{groupTitle}</h2>
+            <h3>{groupTitle}</h3>
```
Comment (jujuzitroni) [inline src/app/components/Card/Card.tsx:33]: is the groupTitle semantically a h3? If not, and you changed it to make the font smaller, consider doing so in css instead.

**Review by jujuzitroni — COMMENTED** [review]
good job! 👍

File: src/app/components/Card/Card.module.css [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
   display: flex;
-  margin: 0 auto;
-  max-width: 30em;
+  margin: 0.5em;
+  width: 80vw;
+  max-width: 40em;
   border-radius: 2em;
   overflow: hidden;
-  box-shadow: 10px 10px 20px rgba(0, 0, 0, 0.2), 0 0 10px rgba(0, 0, 0, 0.2);
+  box-shadow: 8px 5px 10px rgba(0, 0, 0, 0.2), 0 0 5px rgba(0, 0, 0, 0.2);
```
Comment (SGierszewski) [inline src/app/components/Card/Card.module.css:8]: You could also save these specs as a variable for card shadow.

**Review by SGierszewski — COMMENTED** [review]
Well done! Your NavBar is incredible! 🥇

**Review by MaxMuecket — APPROVED** [review]
Great work!

**Pull request merged**